assets/src/stories-editor/constants.js

```diff
@@ -239,3 +239,6 @@ export const AMP_STORY_FONT_IMAGES = {
 
 export const TEXT_BLOCK_BORDER = 5;
 export const TEXT_BLOCK_PADDING = 7;
+
+export const BLOCK_ROTATION_SNAPS = [ -165, -150, -135, -120, -105, -90, -75, -60, -45, -30, -15, 0, 15, 30, 45, 60, 75, 90, 105, 120, 135, 150, 165, 180 ];
```

Comment:
Still something missing. It should maybe have `-180` as well to complete the round.

Reply:
Added in 4d68cfe10a98ae3fafecb59f174d34b7a1913cef. Does that work better now?